code/game/machinery/newscaster/newscaster_machine.dm

```diff
@@ -655,7 +655,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/newscaster, 30)
 			existing_authors += GLOB.news_network.redacted_text
 		else
 			existing_authors += iterated_feed_channel.author
-	if((current_user?.account_holder == "Unknown") || (current_user.account_holder in existing_authors) || isnull(current_user?.account_holder))
+	if((current_user?.account_holder == "Unknown") || (current_user?.account_holder in existing_authors) || isnull(current_user?.account_holder))
```

Comment:
```suggestion
	if(!current_user?.account_holder || current_user.account_holder == "Unknown" || (current_user.account_holder in existing_authors))
```
Just check for a valid account_holder in the first part of the if to simply the consecutive code, looks nicer to me anyway

Reply:
Yeah that's cleaner. Also just had a thought, is the check to see if the name of the account holder is "Unknown" ever relevant? The only situation in which the user's taken name would be unknown is if no ID card was found earlier in the code so the check would fail regardless, right?